code/datums/view.dm

```diff
@@ -118,7 +118,10 @@
 	//Ready for this one?
 	setTo(radius)
 
-/proc/getScreenSize(mob/M) //IMPORTANT: If widescreen toggle preference gets ported, several uses of this proc need to be changed from FALSE to the player pref
-	if(CONFIG_GET(flag/menu_square_view) && M && istype(M, /mob/dead/new_player))
-		return "15x15"	//Return 15x15 for new players because we have normal sized menu screens
-	return CONFIG_GET(string/default_view) //IMPORTANT: If widescreen toggle preference gets ported, this needs to be set to the square view config
+/// Returns the size that the screen should be
+/proc/getScreenSize(mob/M)
+	// New players on the menu get the size of the lobby art
+	if(istype(M, /mob/dead/new_player))
+		return SStitle.lobby_screen_size
+	// Otherwise, they get the default view
+	return CONFIG_GET(string/default_view)
```

Comment:
Off the top of your head do you know if this will mess with people who manually resize their game window during the lobby? Could be an annoyance to them.

Reply:
This will remain functionally the same as what we have now. I think if you have it set to auto-size then it will, if not then it won't